formats/json/commonMain/src/kotlinx/serialization/json/internal/StreamingJsonDecoder.kt

```diff
@@ -343,6 +344,10 @@ internal open class StreamingJsonDecoder(
         }
     }
 
+    override fun decodeStringChunked(consumeChunk: (chunk: String) -> Unit) {
+        lexer.consumeStringChunked(consumeChunk)
```

Comment:
Shouldn't `isLenient` mode be accounted for here?

Reply:
Added support for isLenient mode